google-cloud-storage/src/test/java/com/google/cloud/storage/jqwik/StorageArbitraries.java

```diff
@@ -137,4 +151,48 @@ public String get() {
       return value;
     }
   }
+
+  public static Objects objects() {
+    return Objects.INSTANCE;
+  }
+
+  public static final class Objects {
+    private static final Objects INSTANCE = new Objects();
+
+    private Objects() {}
+
+    public Arbitrary<String> storageClass() {
+      // TODO: return each of the real values and edge cases (including invalid values)
+      return Arbitraries.strings().all().ofMinLength(1).ofLength(1024);
+    }
+
+    public Arbitrary<ObjectChecksums> objectChecksumsArbitrary() {
+      return Combinators.combine(
+              Arbitraries.integers().greaterOrEqual(1),
+              Arbitraries.strings()
+                  .map(
+                      s ->
+                          BaseEncoding.base64()
+                              .encode(Hashing.md5().hashBytes(s.getBytes()).asBytes())))
+          .as(
+              (crc32c, md5) ->
+                  ObjectChecksums.newBuilder()
+                      .setCrc32C(crc32c)
+                      .setMd5Hash(ByteString.copyFrom(md5.getBytes()))
+                      .build());
+    }
+
+    public Arbitrary<CustomerEncryption> customerEncryptionArbitrary() {
+      return Combinators.combine(
+              Arbitraries.strings().ofMinLength(1).ofMaxLength(1024),
+              Arbitraries.strings()
+                  .map(s -> Hashing.sha256().hashString(s, StandardCharsets.UTF_8).asBytes()))
+          .as(
+              (algorithm, key) ->
+                  CustomerEncryption.newBuilder()
+                      .setEncryptionAlgorithm(algorithm)
+                      .setKeySha256Bytes(ByteString.copyFrom(key))
+                      .build());
+    }
+  }
```

Comment:
Nit: move the ByteString copy up in the arbitrary generation
```suggestion
    public Arbitrary<CustomerEncryption> customerEncryptionArbitrary() {
      return Combinators.combine(
              Arbitraries.strings().ofMinLength(1).ofMaxLength(1024),
              Arbitraries.strings()
                  .map(s -> Hashing.sha256().hashString(s, StandardCharsets.UTF_8).asBytes())
                  .map(ByteString::copyFrom))
          .as(
              (algorithm, key) ->
                  CustomerEncryption.newBuilder()
                      .setEncryptionAlgorithm(algorithm)
                      .setKeySha256Bytes(key)
                      .build());
    }

```